docs/Onboarding.md

```diff
@@ -0,0 +1,130 @@
+# BOSS Onboarding Process
+This supplement provides guidance to new development team members and helps them navigate through the steps required to setup their local development environment and deploy a development stack.
+
+A new team member will need to provide them access to the following:
+
+1. An AWS IAM account on [thebossdev](https://thebossdev.signin.aws.amazon.com/console) with credentials that enable aws cli use
+2. Access to the [APL Microns](https://github.com/aplmicrons) repository. The new user will need a bosslet config created and the ability to download the bosslet-configs project.
+3. Access to the [JHUAPL Boss](https://github.com/jhuapl-boss) repository to pull and make code commits.
+
+# Development Envrionment
+The [Installation Guide](./InstallGuide.md) is geared to installing BossDB in new AWS subscription. It covers many of the topics you won't need to get into right now. This **onboarding guide** will highlight what you need to do to get up and running quickly.
+
+## Choose your Python Distribution
+This guide assumes you have installed anaconda for python 3. You can download the installer for your platform from [Anaconda](https://www.anaconda.com/products/individual). 
```

Comment:
How about add verbiage saying that any Python virtual environment may be used?

Reply:
Added Verbiage